pkgs/development/web/bun/default.nix

```diff
@@ -4,25 +4,31 @@ let
     aarch64-darwin = {
       arch = "aarch64";
       shortName = "darwin";
-      sha256 = "sha256-6mi1I8dga16dQLFy2+qa4dzDzlW6J0fdiv104Re3cZ0=";
+      sha256 = "80304f6cf43c6be3db0303bdcb4de4995ace1a394ac6068bbe1e2b6fba32b2e5";
+    };
+
+    aarch64-linux = {
+      arch = "aarch64";
+      shortName = "linux";
+      sha256 = "6b22b6221014fed9e6b6cb432505424e618ef095b2060945ad119cd8f2155fae";
     };
 
     x86_64-darwin = {
       arch = "x64";
       shortName = "darwin";
-      sha256 = "sha256-RGlpwRKLo4Y6uPvwubclIg3wJWePgKTDJvuzdxOrtfM=";
+      sha256 = "33f4c420467af85584ba444606651a0352906c9135d952b266bb6da100ef95bf";
     };
 
     x86_64-linux = {
       arch = "x64";
       shortName = "linux";
-      sha256 = "sha256-Xjm+1wkAsC5Mn6Fm4MRdGyL4gpw2L++N0nKo7ofXLXs=";
+      sha256 = "sha256-UkotflHd2keGeZVSrhwYq45hc7sw8VjybK6Jmn508i8=";
     };
   };
-  dist = dists.${stdenvNoCC.hostPlatform.system} or (throw "Unsupported system: ${stdenvNoCC.hostPlatform.system}");
```

Comment:
Why remove the throw? Its a pretty common pattern with this kind of dist selection

Reply:
Well, bun supports aarch64-Linux now so it's kind of pointless.

Reply:
What about netbsd? What about powerpc? What about windows? What about ARM 32-bit? Please leave it in :smile:.

Reply:
Resolved in the latest commit.

Reply:
The reason behind this is that otherwise you get a syntax error on unsupported platforms which should never happen.